lib/matplotlib/backends/backend_webagg.py

```diff
@@ -113,21 +114,8 @@ class Download(tornado.web.RequestHandler):
         def get(self, fignum, fmt):
             fignum = int(fignum)
             manager = Gcf.get_fig_manager(fignum)
-
-            # TODO: Move this to a central location
-            mimetypes = {
-                'ps': 'application/postscript',
-                'eps': 'application/postscript',
-                'pdf': 'application/pdf',
-                'svg': 'image/svg+xml',
-                'png': 'image/png',
-                'jpeg': 'image/jpeg',
-                'tif': 'image/tiff',
-                'emf': 'application/emf'
-            }
-
-            self.set_header('Content-Type', mimetypes.get(fmt, 'binary'))
-
+            self.set_header(
+                'Content-Type', mimetypes.types_map.get(fmt, 'binary'))
```

Comment:
The fallback should be `application/octet-stream`, should it not?

Reply:
This PR leaves the old behavior, but I'd say that octet-stream is more correct, indeed.